content/ko/docs/contribute/localization_ko.md

```diff
@@ -386,29 +412,33 @@ redirect | 리다이렉트(redirect) |
 redirection | 리다이렉션 |
 Registry | 레지스트리 |
 Release | 릴리스 |
-Replica Set | 레플리카 셋 |
+ReplicaSet | 레플리카셋(ReplicaSet) | API 오브젝트인 경우
 replicas | 레플리카 |
-Replication Controller | 레플리케이션 컨트롤러 |
+ReplicationController | 레플리케이션컨트롤러(ReplicationController) | API 오브젝트인 경우
 repository | 리포지터리 |
+request | 요청(request) | 리소스의 개수나 용량에 대한 요청 수치를 표현하기 위해 사용된 경우 선택적으로 사용 (API 오브젝트 속성으로 request를 사용한 경우는 가능한 영문을 유지)
 resource | 리소스 |
-Resource Limit | 리소스 상한 |
-Resource Quota | 리소스 쿼터 |
+ResourceQuota | 리소스쿼터(ResourceQuota) | API 오브젝트인 경우
 return | 반환하다 |
 revision | 리비전 |
-Role | 롤 |
+Role | 롤(Role) | API 오브젝트인 경우
+RoleBinding | 롤바인딩(RoleBinding) | API 오브젝트인 경우
 rollback | 롤백 |
 rolling update | 롤링 업데이트 |
 rollout | 롤아웃 |
 Romana | 로마나(Romana) |
 Running | Running | 파드의 상태에 한함
 runtime | 런타임 |
+RuntimeClass | 런타임클래스(RuntimeClass) | API 오브젝트인 경우
 Scale | 스케일 |
-Secret | 시크릿 |
+Secret | 시크릿(Secret) | API 오브젝트인 경우
 segment | 세그먼트 |
 Selector | 셀렉터 |
 Self-healing | 자가 치유 |
-Service | 서비스 |
-Service Account | 서비스 어카운트 |
+SelfSubjectAccessReview | 셀프서브젝트액세스리뷰(SelfSubjectAccessReview) | API 오브젝트인 경우
+SelfSubjectRulesReview | SelfSubjectRulesReview | API 오브젝트이지만 용어를 구성하는 단어 중 복수형 Rules를 '룰스'로 외래어 표기하는 경우 한국어 독자에게 다소 생경할 수 있어 예외적으로 영문 용어를 사용함
+Service | 서비스 | API 오브젝트인 경우에도 표현의 간결함을 위해 한영병기를 하지 않음
```

Comment:
I'm surprised not to see
```suggestion
Service | 서비스(Service) | API 오브젝트인 경우에도 표현의 간결함을 위해 한영병기를 하지 않음
```

_but_ it's fine to merge as-is, of course.

Reply:
@sftim For "Pod" and "Service", it was decided not to use Korean-English notation. So, we wrote in the remarks like the below:
"Even in the case of API objects, Korean-English notation are not used for simplicity of expression"